specs/linux/selinux_settings.table

```diff
@@ -0,0 +1,11 @@
+table_name("selinux_settings")
+description("Track active SELinux settings.")
+schema([
+    Column("scope", TEXT, "Scope", index=True),
```

Comment:
Please add more context to the three column descriptions.

Reply:
Thanks for the review! This is mostly a key/value property bag, so I'm not sure what kind of descriptions I could use instead of the current ones. Have you got any suggestion?

Reply:
Try to describe what the term "scope" means to me, assume I know nothing about SELinux. If there are only 3 types of scopes perhaps you can list them in the description.